pkg/cmd/roachtest/github.go

```diff
@@ -32,6 +32,14 @@ type githubIssues struct {
 	teamLoader   func() (team.Map, error)
 }
 
+type issueCategory int
+
+const (
+	clusterCreation issueCategory = iota
```

Comment:
Nit: these are very generic names that will be shared across all roachtest files (as they are all `package main`). We could add an `Error` suffix to these (`sshError`, `otherError`, etc).